src/components/snackbar/snackbar.tsx

```diff
@@ -76,6 +78,12 @@ export class Snackbar {
     @Prop()
     public dismissible: boolean = true;
 
+    /**
+     * Whether to show the snackbar with space for multiple lines of text
+     */
+    @Prop({ reflect: true })
+    public persistent: boolean = false;
```

Comment:
I don't think this is needed, it makes the API a bit ambiguous. The snackbar should be persistent if no timeout has been set

Reply:
isn't it more abstract and confusing to ask the consumer to set some value to `0`, if they don't the snackbar to disappear. Also that needs reading the docs and look at the examples to even understand that this is possible. You can't know it by just looking at the props.

😅 My brain's logic also thinks that if `timeout` is `The amount of time in milliseconds to show the snackbar.` the closer you get to zero, the shorter the snackbar will be displayed. And then if you set to to `0`, it should mean that the snackbar is not displayed at all. 🙃 Am I alone in thinking like this?

Reply:
It's _very_ standard in programming that in cases where _actually zero_ would make no sense, 0 actually means unlimited. That's not something I'd expect the average programmer to have to look up, but if they do, we can just write it in the docblock, and it will show up in their editor's intellisense.

Having a separate property for persistent is ambiguous because what if I set `timeout` to `5000ms` and `persistent` to `true`? Those are contradictory, which is something we don't like in programming 🙂

Reply:
To clarify, if this was user-facing, in the Admin interface for example, I would _absolutely_ argue for having a separate control for `persistent`, like a checkbox, and then either emptying in the timeout value or ideally even hiding the timeout input completely if `persistent` was `true` 🙂

Reply:
Ok, I'll remove it then. 

I always consider these props like UI controls, where toggling a switch (e.g. `persistent`) would "naturally" disable the `timeout` input field. But if `0` makes sense for the right audience (you guys), then we keep it.

p.s.
Checking back the old code we had here, I remember that MDC's original solution was setting the timeout to `-1` to make it persistent.

Reply:
I would say that manually setting timeout to `0` is a bit confusing, I would expect it to just disappear right away even if that makes no sense whatsoever for this component :smile: Setting timeout to `null`, `undefined` or `false` makes more sense to have it persistent, but I don't think we need to have any special handling for the `0` case and just let it be persistent then as well :smile:

Reply:
I guess it's also a bit confusing that the timeout has a default value of `5000`, so if they want it to be persistent they are forced to set a value like `<limel-snackbar timeout={null} />`